tests/oss-fuzz/pdu_parse_target.c

```diff
@@ -0,0 +1,11 @@
+#include <coap.h>
+
+int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
```

Comment:
I would prefer to follow the libcoap coding guidelines here, i.e., put the return type on a single line and
start the next line with the function identifier and end with the open bracket for the function block:

```c
int
LLVM....() {
```